app/Policies/GamePolicy.php

```diff
@@ -156,6 +156,22 @@ public function manageContributionCredit(User $user, Game $game): bool
         ]);
     }
 
+    public function viewDeveloperInterest(User $user, Game $game): bool
+    {
+        // Developers need to have a claim on the game in order to see
+        // who else might have expressed interest in working on it.
+        // This is mainly to facilitate them reaching out to collab.
+        if ($user->hasRole(Role::DEVELOPER) && $user->hasActiveClaimOnGameId($game->id)) {
```

Comment:
The logic to get to this page allows any primary claimaint:
https://github.com/RetroAchievements/RAWeb/blob/21244ca9ee78d57ff64f318e5bad2b51e653333d/resources/views/pages-legacy/gameInfo.blade.php#L587

This blocks Jrs (which do have a link), and allows collaborators (which don't have a link).

Reply:
Fixed in latest & added another test case.